src/common/HaskellDo/Toolbar/View.hs

```diff
@@ -62,8 +65,20 @@ openProjectModal =
         div ! atr "class" "modal-footer" $
             div ! id "closeModalButton" $ noHtml
 
+convertToPDFModal :: Perch
+convertToPDFModal =
+    div ! id "convertToPDFModal" ! atr "class" "modal" $ do
+        div ! atr "class" "modal-content" $ do
+          h4 ("PDF saved on project path" :: String)
+
+convertToPDFModalFail :: Perch
+convertToPDFModalFail =
+    div ! id "convertToPDFModalFail" ! atr "class" "modal" $ do
+        div ! atr "class" "modal-content" $ do
+          h4 ("Error: wkhtmltopdf is not installed or a project has not been loaded" :: String)
```

Comment:
I tried to look for other Materialize components but I didn't found a better one for the messages 😮